### PR TITLE
Connect created presentational components with routing - release/v2.1

### PR DESCRIPTION
**Pull request by kowalowy1234**

### CODE REVIEW

File: billennium/apps/billespace/src/app/app.provider.tsx [hunk 1]
```diff
@@ -13,9 +12,7 @@ export const AppProvider = ({ children }: AppProviderProps) => {
     <MenuProvider>
       <ScreenSlideProvider>
         <OfficeManagementProvider>
-          <MakeReservationProvider>
-            <ReservationProvider>{children}</ReservationProvider>
-          </MakeReservationProvider>
+          <MakeReservationProvider>{children}</MakeReservationProvider>
```
Comment (ajronn) [inline billennium/apps/billespace/src/app/app.provider.tsx:15]: Write it by 3 lines

File: billennium/apps/billespace/src/app/app.tsx [hunk 2]
```diff
@@ -3,8 +3,9 @@ import { Layout, Topbar } from '@billennium/ui';
 import Loadable from 'react-loadable';
 import { Navigation } from './modules/navigation';
 import { AppProvider } from './app.provider';
-import Login from './modules/login';
 import { LoginGuard } from './loginGuard';
+import HomeScreen from './modules/home';
+import { LINKS } from './utils';
 
 const Loading = (props: { error: string | undefined }) => {
```
Comment (ajronn) [inline billennium/apps/billespace/src/app/app.tsx:10]: extract it into a separate file

File: billennium/apps/billespace/src/app/modules/home/home.module.tsx [hunk 3]
```diff
@@ -0,0 +1,6 @@
+import { Home } from './components';
+
+export default function HomeScreen() {
```
Comment (ajronn) [inline billennium/apps/billespace/src/app/modules/home/home.module.tsx:3]: dont use default export

File: billennium/apps/billespace/src/app/modules/reservations/reservations.module.tsx [hunk 4]
```diff
@@ -1,8 +1,18 @@
-import { useContext } from 'react';
-import { ReservationContext } from './reservations.provider';
+import { ReactNode, useState } from 'react';
+import { ReservationsSummary } from './components';
+
+const STAGES = ['RESERVATION_SUMMARY'] as const;
```
Comment (ajronn) [inline billennium/apps/billespace/src/app/modules/reservations/reservations.module.tsx:4]: try enum here

File: billennium/apps/billespace/src/app/utils/links.tsx [hunk 5]
```diff
@@ -0,0 +1,12 @@
...
+  OFFICE_MANAGEMENT: 'office-management',
+  RESERVATION_HISTORY: 'reservation-history',
+  MAKE_RESERVATION: 'make-reservation',
+  LAST_RESERVATIONS: 'last-reservations',
+  RESERVATION_DETAILS: 'reservation-details',
+  OFFICE_DETAILS: 'office-details',
+  RESERVATIONS: 'reservations',
+  OFFICES: 'offices',
+  BACK: '..',
+};
```
Comment (kowalowy1234) [inline billennium/apps/billespace/src/app/utils/links.tsx:12]: make it so it's not mutable

**Pull request merged**